CMakeLists.txt

```diff
@@ -34,3 +34,5 @@ add_subdirectory(idl)
 
 add_subdirectory(projects)
 add_subdirectory(src)
+
+option(REAL_ROBOT "If this is ON, mc_openrtm will attempt to change servo PD gains through IOB." OFF)
```

Comment:
I'd prefer this to be a run-time switch. We can easily detect if mc_openrtm is being run on a real robot (`RobotHardware` component is available) or in simulation